### ISSUE TITLE
Division by zero (PrettyEmbedHelper\Eel\Helper::paddingTop($ratio = null))

### ISSUE THREAD

**Issue by wolfgang-stark** [jonnitto/Jonnitto.PrettyEmbedVideoPlatforms#4]
Hi jonnitto,

thank you for this nice plugin!

Embedding a portrait Youtube video lead to an 500 error.

According to call stack

```
237 Neos\Eel\CompilingEvaluator_Original::evaluate("Jonnitto.PrettyEmbedHelper.paddingTop(this.ratio || this.force16to9)", Neos\Eel\ProtectedContext)
236 Neos\Eel\Utility_Original::evaluateEelExpression("${Jonnitto.PrettyEmbedHelper.paddingTop(this.ratio || this.force16to9)}", Neos\Eel\CompilingEvaluator, array|37|)
```

It looks like the bool `false` gets passed to paddingTop https://github.com/jonnitto/Jonnitto.PrettyEmbedHelper/blob/master/Classes/Eel/Helper.php#L25 

And paddingTop doesn't handle this case, casts `false` to `0` and triggers `Division by zero`.

Is this a bad configuration or a bug?

Config:

```yml
  PrettyEmbedVideoPlatforms:
    defaults:
      lightbox: false
      preserveAspectRatio: false
      force16to9: false
```

Thank you very much!

**Comment by jonnitto** [comment]
This is a bug. Do you want to open a PR?

**Comment by jonnitto** [comment]
Closed with #3
